src/vs/workbench/services/configurationResolver/test/electron-browser/configurationResolverService.test.ts

```diff
@@ -77,6 +82,26 @@ suite('Configuration Resolver Service', () => {
 		}
 	});
 
+	test('workspace folder with argument', () => {
+		if (platform.isWindows) {
+			assert.strictEqual(configurationResolverService!.resolve(workspace, 'abc ${workspaceFolder:workspaceLocation} xyz'), 'abc \\VSCode\\workspaceLocation xyz');
+		} else {
+			assert.strictEqual(configurationResolverService!.resolve(workspace, 'abc ${workspaceFolder:workspaceLocation} xyz'), 'abc /VSCode/workspaceLocation xyz');
+		}
+	});
+
+	test('workspace folder with undefined workspace folder', () => {
+		assert.throws(() => configurationResolverService!.resolve(undefined, 'abc ${workspaceFolder} xyz'));
```

Comment:
Can you add another test that tests with an invalid argument, such as "invalidLocation" instead of "workspaceLocation"?

Reply:
Same for relativeFile

Reply:
Done.